README.md

```diff
@@ -108,7 +108,7 @@ g.index.writable?
 g.repo
 g.dir
 
-g.log   # returns array of Git::Commit objects
+g.log   # returns Enumberator of Git::Commit objects
```

Comment:
"Enumberator" is misspelled.

Maybe it would be better to say: "returns a Git::Log object which is an Enumerator of Git::Commit objects"

Reply:
I thought I had fixed the spelling error :(

Your suggestion is an improvement and should be used.

Reply:
If you agree with my suggestion, please redo this commit and I'll make sure it is reviewed and merged.

Reply:
Just did that

Reply:
Squished